packages/chart/src/components/LinearChart.jsx

```diff
@@ -667,19 +672,25 @@ const LinearChart = props => {
           {['Line', 'Bar', 'Combo', 'Area Chart'].includes(config.visualizationType) && !isHorizontal && <ZoomBrush xScaleBrush={xScaleBrush} yScale={yScale} xMax={xMax} yMax={yMax} />}
           {/* Line chart */}
           {/* TODO: Make this just line or combo? */}
-          {visualizationType !== 'Bar' && visualizationType !== 'Paired Bar' && visualizationType !== 'Box Plot' && visualizationType !== 'Area Chart' && visualizationType !== 'Scatter Plot' && visualizationType !== 'Deviation Bar' && visualizationType !== 'Forecasting' && (
-            <>
-              <LineChart xScale={xScale} yScale={yScale} getXAxisData={getXAxisData} getYAxisData={getYAxisData} xMax={xMax} yMax={yMax} seriesStyle={config.series} />
-            </>
-          )}
+          {(visualizationType !== 'Bar' || displayLineGraphAs !== 'Bar') &&
+            visualizationType !== 'Paired Bar' &&
+            visualizationType !== 'Box Plot' &&
+            visualizationType !== 'Area Chart' &&
+            visualizationType !== 'Scatter Plot' &&
+            visualizationType !== 'Deviation Bar' &&
+            visualizationType !== 'Forecasting' && (
```

Comment:
I know you didn't write all this but could you try refactoring this to 

!['Paired Bar', 'Box Plot', 'Area Chart', ....].includes(visualizationType)